README.md

```diff
@@ -333,6 +333,47 @@ With this flag the extra value will be parsed normally.
 
 **USE AT YOUR OWN RISK!**
 
+### `void llhttp_set_lenient_version(llhttp_t* parser, int enabled)`
+
+Enables/disables lenient handling of HTTP version.
+
+Normally `llhttp` would error when the HTTP version in the request or status line
+is not `0.9`, `1.0`, `1.1` or `2.0`.
+With this flag the extra value will be parsed normally.
+
+**(USE AT YOUR OWN RISK)**
```

Comment:
I'd rephrase it to something more specific... like, if you are relying on this parsing, don't use this function from a security point of view. Or mention some CWE it implies.

Reply:
What about now?